tests/integration/test_bundle_deployment.py

```diff
@@ -10,15 +10,19 @@
 from lightkube.resources.core_v1 import Service
 
 BUNDLE_NAME = "kubeflow"
+BUNDLE_VERSION = os.environ.get("BUNDLE_VERSION")
+
 
 @pytest.fixture()
 def lightkube_client() -> lightkube.Client:
     client = lightkube.Client(field_manager=BUNDLE_NAME)
     return client
 
+
 @pytest.fixture
 def bundle_path() -> str:
-    return os.environ.get("BUNDLE_PATH").replace("\"", "")
+    return f"./releases/{BUNDLE_VERSION}/stable/kubeflow/bundle.yaml"
```

Comment:
this is indeed cleaner than before, I have a concern tho that in case we need to run a different risk for releases in progress we will not be able to

for example when we pointed to 1.9 tests to beta in https://github.com/canonical/bundle-kubeflow/pull/986/files#diff-ef2cef9f88b4fe09ca3082140e67f5ad34fb65fb6e228f119d3812261ae51449R89

Reply:
> should we have version and risk, perhaps?

yea that's a good idea, but for latest/edge we have to either treat it seperately or fix the directory structure to be the same as the other release folders as @orfeas-k is suggesting ^

Reply:
I have changed the approach slightly, please let me know what you think.

Reply:
Resolving as this does not apply any more.